declarations.d.ts

```diff
@@ -220,10 +220,18 @@ interface IAnalyticsService {
 	track(featureName: string, featureValue: string): IFuture<void>;
 }
 
+interface IAllowEmpty {
+	allowEmpty?: boolean
```

Comment:
why is this in a separate interface, can't you add it as an optional property in IPrompterOptions?